app/models/miq_server/configuration_management.rb

```diff
@@ -2,17 +2,12 @@ module MiqServer::ConfigurationManagement
   extend ActiveSupport::Concern
   include ConfigurationManagementMixin
 
-  def get_config(type = "vmdb")
-    if is_local?
-      VMDB::Config.new(type)
-    else
-      VMDB::Config.for_resource(type, self)
-    end
+  def settings
```

Comment:
I'm not sure I understand the concern. This is a `settings` method on `MiqServer` only, why would it matter if there is a `Zone#settings`? Or are you just confused because they would point to different things?

Reply:
yeah they would be different, but both (+MiqRegion) are "things from which you can get Settings"

Reply:
there is no code conflict...more of a conceptual conflict.  Again, it might be ok.